react-game/src/components/AllTrials.js

```diff
@@ -0,0 +1,69 @@
+import { useState, useEffect } from 'react';
```

Comment:
we should import React from "react"; in react components

Reply:
done